test/components/countdownManager-test.js

```diff
@@ -8,16 +8,13 @@ var sinon = require('sinon');
 
 describe('Countdown Manager', function() {
 
-  var timer = null;
+  var clock = null;
```

Comment:
I would change this back to timer (unless timer is used for something else I missed), make your changes look smaller